Framework/API/src/DetectorInfo.cpp

```diff
@@ -32,6 +34,18 @@ DetectorInfo::DetectorInfo(
     m_detIDToIndex[m_detectorIDs[i]] = i;
 }
 
+/// Assigns the contents of the non-wrapping part of `rhs` to this.
```

Comment:
Is this what you really expect from an assignment? It's not immediately clear to me why this is being done like this.